docs/getting-started/quickstart.md

```diff
@@ -626,7 +643,7 @@ INFO:     Uvicorn running on http://0.0.0.0:8888 (Press CTRL+C to quit)
 
 ![](../reference/ui.png)
 
-## Step 5: Re-examine `test_workflow.py`
+## Step 10: Re-examine `test_workflow.py`
```

Comment:
I think the originally numbering had a structure that outlined that some components were optional or sub-steps so we should probably modify these to reflect that. For example, the previous Step 3 had a bunch of sub-steps to reflect that hierarchy.

Reply:
I know what you are saying but one of the comments (number 8) in https://github.com/feast-dev/feast/issues/4593 asks for the changes that I did